.travis.yml

```diff
@@ -1,9 +1,5 @@
 language: python
 
-branches:
-  only:
-  - master
```

Comment:
This change caused tests to be run twice for this PR :)
Travis runs tests for PRs regardless of branch configuration.

Reply:
Got it..
I was testing some stuff, as this repository wasn't showing there :)